docs/how-to-guides/upgrade-openebs.md

```diff
@@ -0,0 +1,303 @@
+from `v1.12.0` to `v2.2.0` and migrate from cStor Pools/Volumes to latest CSI based provisioning
+(CSPC Pools/Volumes).
+
+## Prerequisites
+
+To upgrade the OpenEBS data plane components, we need the following:
+
+* Lokomotive `v0.6.0`
+    ```bash
+    lokoctl version
+    v0.6.0
+    ```
+
+* A Lokomotive cluster accessible via `kubectl` deployed on a [Equinix]
+    metal(https://metal.equinix.com/).
+
+* OpenEBS `v1.12.0` installed. You can check if OpenEBS is indeed in expected
+    version:
+    ```bash
+    $ kubectl get pods -n openebs -n openebs.io/version=1.12.0
+    ```
+
+* Highly recommended to schedule a downtime for the applications consuming the
+    OpenEBS PV and make sure to take a backup of the data before starting the
+    below upgrade procedure.
+
+
+## Steps
+
+###Step 1: Upgrade OpenEBS control plane components
+
+Lokomotive provides an easy way of upgrading the OpenEBS control plane.
+
+Execute the following command to upgrade OpenEBS control plane:
+
+```bash
+lokoctl component apply openebs-operator
+```
+
+Doing so, terminates the OpenEBS resources associated with `v1.12.0` and new
+resources are created with `v2.2.0`.
+
+Verify all the pods are in `Running` state before proceeding:
+
+```bash
+$ kubectl get pods -n openebs -n openebs.io/version=2.2.0
+```
+
+### Step 2: Upgrade OpenEBS data plane components
+
+OpenEBS control plane and data plane components work independently. Even if the
+control plane components are upgraded to `v2.2.0`, the data plane components
+continue to work with the older version `v1.12.0`.
+
+#### Upgrade cStor pools
+
+Get existing `StoragePoolClaims` by executing:
+
+```bash
+kubectl get spc
+NAME                           AGE
+cstor-pool-openebs-replica-3   2d21h26m
+```
+
+Create a Job to upgrade the existing cStor pools:
+
+```bash
+# upgrade-cstor-pools-1120-220.yaml
+
+#This is an example YAML for upgrading cstor SPC.
+#Some of the values below needs to be changed to
+#match your openebs installation. The fields are
+#indicated with VERIFY
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  #VERIFY that you have provided a unique name for this upgrade job.
+  #The name can be any valid K8s string for name. This example uses
+  #the following convention: cstor-spc-<flattened-from-to-versions>
+  name: cstor-spc-1120220
+
+  #VERIFY the value of namespace is same as the namespace where openebs components
+  # are installed. You can verify using the command:
+  # `kubectl get pods -n <openebs-namespace> -l openebs.io/component-name=maya-apiserver`
+  # The above command should return status of the openebs-apiserver.
+  namespace: openebs
+spec:
+  backoffLimit: 4
+  template:
+    spec:
+      #VERIFY the value of serviceAccountName is pointing to service account
+      # created within openebs namespace. Use the non-default account.
+      # by running `kubectl get sa -n <openebs-namespace>`
+      serviceAccountName: openebs-operator
```

Comment:
I think this comment can be removed since we know what the service account is called.

Reply:
Or as lokomotive we control what it is called.